src/util.cc

```diff
@@ -1418,6 +1418,42 @@ TempFile create_temporary_file(const char* pattern) {
   return result;
 }
 
+static ScopedFd create_memfd_file(const string &real_name) {
+  ScopedFd fd(syscall(SYS_memfd_create, real_name.c_str(), 0));
+  return fd;
+}
+
+static void replace_char(string& s, char c, char replacement) {
+  size_t i;
+  while (string::npos != (i = s.find(c))) {
+    s[i] = replacement;
+  }
```

Comment:
Isn't this O(N^2)?

Reply:
Yes, but this code just moved from EmuFs.cc ;). I can fix it though.

Reply:
Sorry!